### PR TITLE
Add normalize.css

### PR DESCRIPTION
**Pull request by Ibochkarev**
### What does it do?
Add normalize.css

### Why is it needed?
Normalize.css is an alternative to CSS resets. 

**The aims of normalize.css are as follows:**
- The aims of normalize.css are as follows:
- Preserve useful browser defaults rather than erasing them.
- Normalize styles for a wide range of HTML elements.
- Correct bugs and common browser inconsistencies.
- Improve usability with subtle improvements.
- Explain the code using comments and detailed documentation.
- It supports a wide range of browsers (including mobile browsers) and includes CSS that normalizes HTML5 elements, typography, lists, embedded content, forms, and tables.

Despite the project being based on the principle of normalization, it uses pragmatic defaults where they are preferable.

### Related issue(s)/PR(s)
#12991

### CODE REVIEW

**Comment by Jako** [comment]
This will not help with #12991, since it does not set the default input color/background.

**Comment by Ibochkarev** [comment]
@Jako If this does not help with this problem, it will still improve the correctness of displaying the default styles of different browsers into a single view.

**Comment by Jako** [comment]
Things like this should be started in 3.x. It is an enhancement, that could have an impact for the whole manager styling.

**Comment by Ibochkarev** [comment]
~~I will close this PR. Add Normalize.css, together with a complete refactoring of the styles and corrections in the administrative panel in the branch 3.x.~~

**Comment by Ibochkarev** [comment]
@Jako I changed the branch to 3.x. Please check.

File: _build/templates/default/package.json [hunk 1]
```diff
@@ -21,6 +21,9 @@
   "dependencies": {
     "bourbon": "^5.1.0",
     "bourbon-neat": "^3.0.0",
-    "@fortawesome/fontawesome-free": "^5.7.0"
+    "@fortawesome/fontawesome-free": "^5.7.0",
+    "font-awesome": "^4.7.0",
```
Comment (Jako) [inline _build/templates/default/package.json]: `"font-awesome": "^4.7.0"` has to be removed.

File: _build/templates/default/package.json [hunk 2]
```diff
@@ -21,6 +21,9 @@
...
+    "node-neat": "^1.7.2",
```
Comment (Jako) [inline _build/templates/default/package.json]: This line should be there, when 2.x is merged into 3.x

File: _build/templates/default/sass/index.scss [hunk 3]
```diff
@@ -22,6 +22,7 @@ $fa-css-prefix: icon;
 
 $fa-css-prefix: fa;
 
+@import "normalize/_normalize";
```
Comment (Jako) [inline _build/templates/default/sass/index.scss]: You have added the normalize-scss package to package.json but you have included the files of that package in this commit too (and this twice: i.e. normalize/normalize/_variables.scss and normalize/_variables.scss!). Adding the files is superfluous.

Please refer here to the files in this line by descending/ascending the tree with multiple `../` until you get to `node_modules/normalize-scss/sass/_normalize.scss` (please check the path).

**Review by Jako — CHANGES_REQUESTED** [review]

File: _build/templates/default/sass/index.scss (hunk 3 above)
Comment (Ibochkarev) [reply _build/templates/default/sass/index.scss]: In the gruntfile.js I made similar to Font-awesome copying files from node_modules folder. Next is the connection of one normalize file.scss . The way I fixed it.

Comment (Jako) [reply _build/templates/default/sass/index.scss]: But copying is not needed, since there are no files referenced by url like font files for the browser.

Comment (Jako) [reply _build/templates/default/sass/index.scss]: Copying the scss files of fontawesome is too much too. The location of the font files could be set with a sass variable (if thats not done already) and the scss files could be included directly from the node_modules folder.

Comment (Ibochkarev) [reply _build/templates/default/sass/index.scss]: Fixed

Comment (Jako) [reply _build/templates/default/sass/index.scss]: Thanks!

**Review by Jako — APPROVED** [review]

File: _build/templates/default/sass/index.scss [hunk 4]
```diff
@@ -1,3 +1,4 @@
+@import "../node_modules/normalize-scss/fork-versions/default/normalize";
```
Comment (Jako) [inline _build/templates/default/sass/index.scss]: according to the [docs](https://github.com/JohnAlbin/normalize-scss) this should be used (Approach 2 – no mixin): 

`@import "[path to]/normalize-scss/sass/normalize/import-now"`

**Review by Jako — CHANGES_REQUESTED** [review]

File: _build/templates/default/sass/index.scss (hunk 4 above)
Comment (Jako) [reply _build/templates/default/sass/index.scss]: The fork-versions are meant to be edited after copying.

Comment (Ibochkarev) [reply _build/templates/default/sass/index.scss]: @Jako Thanks

**Review by Jako — APPROVED** [review]

**Comment by Mark-H** [comment]
The reported issue was closed as no longer being a problem so I wonder if this is a necessary change and if this isn't going to cause new issues due to changing defaults?

**Comment by muzzwood** [comment]
That specific issue was closed as Firefox changed how they did their rendering.
However, as it may be an issue again in the future, this could be worth doing now if it doesn't cause too many problems.
I don't mind at all if it's held off until there is another problem though. It's not a big priority for now.

**Comment by OptimusCrime** [comment]
I am in favor of introducing a normalize stylesheet, even if it does not solve a problem here and now. Having a default that is not vendor specific makes life easier for everyone.

**Comment by Ibochkarev** [comment]
@OptimusCrime thanks my friend!

**Review by Mark-H — APPROVED** [review]
Fair enough. I just hope it doesn't break ExtJS styles that depend on legacy behaviour ;)

**Pull request merged**